### ISSUE TITLE
Using Costura 1.4.1 on .Net 3.5 target project fails build

### ISSUE THREAD

**Issue by mzboray** [Fody/Costura#199]
I am getting this error in a new .NET 3.5 project using Costura 1.4.1 and Fody 2.0.7 set to embed Newtonsoft.Json 10.0.2:

1>------ Rebuild All started: Project: FodyTest2, Configuration: Debug Any CPU ------
1>    Fody: Fody (version 2.0.7.0) Executing
1>      Fody/Costura:   	Embedding 'c:\Users\mzboray\Documents\Visual Studio 2017\Projects\FodyTest2\packages\Newtonsoft.Json.10.0.2\lib\net35\Newtonsoft.Json.dll'
1>MSBUILD : error : Fody: An unhandled exception occurred:
1>MSBUILD : error : Exception:
1>MSBUILD : error : Value cannot be null.
1>MSBUILD : error : Parameter name: method
1>MSBUILD : error : StackTrace:
1>MSBUILD : error :    at Mono.Cecil.Mixin.CheckMethod(Object method)
1>MSBUILD : error :    at Mono.Cecil.ModuleDefinition.ImportReference(MethodReference method, IGenericParameterProvider context)
1>MSBUILD : error :    at ModuleWeaver.Import(Object operand) in C:\projects\costura\src\Costura\AssemblyLoaderImporter.cs:line 273
1>MSBUILD : error :    at ModuleWeaver.CloneInstruction(Instruction instruction) in C:\projects\costura\src\Costura\AssemblyLoaderImporter.cs:line 227
1>MSBUILD : error :    at ModuleWeaver.CopyInstructions(MethodDefinition templateMethod, MethodDefinition newMethod) in C:\projects\costura\src\Costura\AssemblyLoaderImporter.cs:line 210
1>MSBUILD : error :    at ModuleWeaver.CopyMethod(MethodDefinition templateMethod, Boolean makePrivate) in C:\projects\costura\src\Costura\AssemblyLoaderImporter.cs:line 153
1>MSBUILD : error :    at ModuleWeaver.ImportAssemblyLoader(Boolean createTemporaryAssemblies) in C:\projects\costura\src\Costura\AssemblyLoaderImporter.cs:line 62
1>MSBUILD : error :    at ModuleWeaver.Execute() in C:\projects\costura\src\Costura\ModuleWeaver.cs:line 33
1>MSBUILD : error :    at lambda_method(Closure , Object )
1>MSBUILD : error :    at InnerWeaver.ExecuteWeavers() in C:\projects\fody\FodyIsolated\InnerWeaver.cs:line 180
1>MSBUILD : error :    at InnerWeaver.Execute() in C:\projects\fody\FodyIsolated\InnerWeaver.cs:line 85
1>MSBUILD : error : Source:
1>MSBUILD : error : Mono.Cecil
1>MSBUILD : error : TargetSite:
1>MSBUILD : error : Void CheckMethod(System.Object)
1>MSBUILD : error : 
1>    Fody:   Finished Fody 20ms.
========== Rebuild All: 0 succeeded, 1 failed, 0 skipped ==========

The target dll to include is Newtonsoft.Json but I don't think that really matters.

**Comment by pushqrdx** [comment]
Same error here

```
Severity	Code	Description	Project	File	Line	Suppression State
Error		Fody: An unhandled exception occurred:
Exception:
Value cannot be null.
Parameter name: method
StackTrace:
   at Mono.Cecil.Mixin.CheckMethod(Object method)
   at Mono.Cecil.ModuleDefinition.ImportReference(MethodReference method, IGenericParameterProvider context)
   at ModuleWeaver.Import(Object operand) in C:\projects\costura\src\Costura\AssemblyLoaderImporter.cs:line 272
   at ModuleWeaver.CloneInstruction(Instruction instruction) in C:\projects\costura\src\Costura\AssemblyLoaderImporter.cs:line 226
   at ModuleWeaver.CopyInstructions(MethodDefinition templateMethod, MethodDefinition newMethod) in C:\projects\costura\src\Costura\AssemblyLoaderImporter.cs:line 209
   at ModuleWeaver.CopyMethod(MethodDefinition templateMethod, Boolean makePrivate) in C:\projects\costura\src\Costura\AssemblyLoaderImporter.cs:line 152
   at ModuleWeaver.ImportAssemblyLoader(Boolean createTemporaryAssemblies) in C:\projects\costura\src\Costura\AssemblyLoaderImporter.cs:line 61
   at ModuleWeaver.Execute() in C:\projects\costura\src\Costura\ModuleWeaver.cs:line 35
   at lambda_method(Closure , Object )
   at InnerWeaver.ExecuteWeavers() in C:\projects\fody\FodyIsolated\InnerWeaver.cs:line 181
   at InnerWeaver.Execute() in C:\projects\fody\FodyIsolated\InnerWeaver.cs:line 86
Source:
Mono.Cecil
TargetSite:
Void CheckMethod(System.Object)	Detroit.WindowsService			
```

**Comment by distantcam** [comment]
This should be fixed in 1.6.2. Thanks.
